addons/marker_flags/functions/fnc_pickUpFlag.sqf

```diff
@@ -0,0 +1,25 @@
+#include "script_component.hpp"
+/*
+ * Author: Brett Mayson
+ * Places a flag in front of the unit.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ * 1: Flag <OBJECT>
+ *
+ * Return Value:
+ * Nothing
+ *
+ * Example:
+ * [player, _flag] call ace_marker_flags_fnc_pickupFlag
+ *
+ * Public: No
+ */
+
+params [["_unit", objNull, [objNull]], ["_flag", objNull, [objNull]]];
+
+if (isNull _flag) exitWith {};
+
+[_unit, "PutDown"] call EFUNC(common,doGesture);
+deleteVehicle _flag;
+[_unit, configName configOf _flag] call EFUNC(common,addToInventory);
```

Comment:
This is a bit weird. The item has a config entry that defines what object is placed while the object (when picked up) assumes that the item's config name is the same as its own.

Reply:
good catch, should be fixed via 6e56e4f307fc1184d5bf8c4dc17a817f2a676620